src/bin/ch-remote.rs

```diff
@@ -1022,6 +1050,20 @@ fn get_cli_commands_sorted() -> Box<[Command]> {
                     .help("New memory size in bytes (supports K/M/G suffix)")
                     .num_args(1),
             ),
+        Command::new("resize-disk")
+            .about("grows/shrinks an attached disk")
```

Comment:
Maybe this should be `.help()`

Reply:
My mistake - I was reading the wrong context. I think maybe it should be "Resize attached disk"